src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -660,11 +660,15 @@ private void removeTerritoriesWhereTransportsAreExposed() {
             ProBattleUtils.calculateBattleResults(player, t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data),
                 patd.getBombardTerritoryMap().keySet(), true);
         int production = 0;
+        int isEnemyCapital = 0;
         final TerritoryAttachment ta = TerritoryAttachment.get(t);
         if (ta != null) {
           production = ta.getProduction();
+          if (ta.isCapital()) {
+            isEnemyCapital = 1;
+          }
         }
-        final double attackValue = result.getTUVSwing() + production;
+        final double attackValue = result.getTUVSwing() + production * (1 + 3 * isEnemyCapital);
```

Comment:
- magic numbers code smell. The "1" and "3" are pretty meaningless to me, and likely will be to the author as well in a few months time

- lack of regression tests. It really does seem like you could mock out some of the key data structures and build tests around this logic. Then there would be confidence we are not hurting some other corner case.